docs/howtos/DataPlane Generation - DPG/09versioning.mdx

```diff
@@ -0,0 +1,207 @@
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Versioning
+
+This doc details what emitters will generate for versioned specs
+
+## Single API Version
+
+If there is just one API version in the spec, we will generate
+
+## Multiple API version
+
+The configuration flag `api-version` allows you to toggle the behavior that our emitters will generate.
+
+We will get the versioning information from the `Versions` enum that you define and link as part of the `@typespec/versioning` library.
+
+> NOTE: The ordering of the values in the `Versions` enum is very important. We use this information to determine the order of versions. Our default value will be the **last entry** in the `Versions` list
+
+### Default
+
+By default our emitters will only generate the surface used by the latest api version if there are multiple defined.
+
+By default, only models used in api surface of the latest api version are included in generation. Documentation and enums showing the available API versions will still include all of the known api versions, meaning there will be documentation for both the preview and stable releases.
```

Comment:
Maybe better to put these statements together into one block, like:
```suggestion
By default our emitters will only generate the surface used by the latest api version if there are multiple defined. This includes generating only the models used in the surface area of the latest api version. 

Documentation and enums showing the available API versions will still include all of the known api versions, meaning there will be documentation for both the preview and stable releases.
```

Reply:
Also, this is a style nit but should we be consistent about the casing of API?

Reply:
fair, let me always lowercase it